packages/@aws-cdk/aws-rds/package.json

```diff
@@ -64,6 +64,7 @@
   "devDependencies": {
     "@aws-cdk/assert": "0.0.0",
     "@aws-cdk/aws-events-targets": "0.0.0",
+    "@aws-cdk/aws-lambda": "0.0.0",
```

Comment:
Why is this needed as a dev dependency?

Reply:
for testing purposes as there are a couple of non-logretention-related tests that use lambda e.g.

https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-rds/test/test.instance.ts#L340

https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-rds/test/integ.instance.lit.ts#L85